src/System.Private.ServiceModel/tools/CertificateCleanup/CertificateCleanup.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Nit: this might belong under tools\test.  I know it's a little ambiguous right now, but I was thinking tools might be reserved for things like msbuild tools or tools that we actually ship.  My thinking was that tools\test indicated these tools were required only for testing.  Opinions welcome how to organize tools folder.